lib/widget/camera.dart

```diff
@@ -135,6 +158,23 @@ class CameraScreenState extends WidgetState<CameraScreen> with WidgetsBindingObs
     setState(() {});
   }
 
+  void defineErrorText() {
+    if (widget._controller.mode == CameraMode.photo) {
+      errorString =
+          'Maximum ${widget._controller.maxCount} images may be selected';
+      cameraoptionsList = ['PHOTO'];
```

Comment:
why are you setting cameraoptionsList in a function that is just constructing the error message? this function should not change any state right? it's just a getter for the message

Reply:
I’m setting camera option list value in this function because i called this function in initstate and it sets message as well as camera option list value will be equal to mode. I think we should change the function name. That’s your call. I’ll do accordingly